src/resources/PlatformResources.ts

```diff
@@ -119,6 +121,7 @@ class PlatformResources {
     index: Index;
     insightPanelConfig: InsightPanelConfig;
     insightPanelInterface: InsightPanelInterface;
+    ipxInterface: IPXInterface;
```

Comment:
Would it make sense to put it inside the existing `ipx` resource? So that the user would call `Platform.ipx.interface.list`.  

I know nothing about the feature, so if what I am suggesting doesn't make sense to you, ignore this comment 😄

Reply:
Not a bad idea, but I think it makes more sense to keep it named `ipxInterface` since it better reflects that it is an interface that is coming from the Search Interface Service, like the insight panel. 
Plus, when we will delete the old `IPX` resource, it won't make sense to have `ipx.ipxinterface.list` imo